docs/source/data/subscriptions.mdx

```diff
@@ -197,13 +197,23 @@ httpServer.listen(PORT, () => {
 
 </ExpansionPanel>
 
+> ⚠️ **Running into an error?** If you are using the `graphql-ws` library your specified subscription protocol must be consistent across your backend, frontend and **every other tool** you use (including [Apollo Sandbox](../studio/explorer/additional-features/#subscription-support)). For more information see [Switching from `subscriptions-transport-ws`](#switching-from-subscriptions-transport-ws).
+
 ## Resolving a subscription
 
 Resolvers for `Subscription` fields differ from resolvers for fields of other types. Specifically, `Subscription` field resolvers are **objects** that define a `subscribe` function:
 
 ```ts title="index.ts"
 const resolvers = {
   Subscription: {
+    hello: {
+      // Example using an async generator
+      subscribe: async function* () {
+        for await (const word of ["Hello", "Bonjour", "Ciao"]) {
+          yield { hello: word };
```

Comment:
assuming you tested this :)

Reply:
Yes, I did—I also learned way more about async generators while writing this! 🧠